src/main/groovy/dev/jbang/gradle/tasks/JBangTask.groovy

```diff
@@ -336,20 +336,16 @@ class JBangTask extends DefaultTask {
     }
 
     private void unzip(File zipFile, Path installDir) throws IOException {
-        ZipArchiveInputStream archive = null
-        try {
-            archive = new ZipArchiveInputStream(new BufferedInputStream(new FileInputStream(zipFile)))
-
-            ZipArchiveEntry entry = null
+        try (archive = new ZipArchiveInputStream(new BufferedInputStream(new FileInputStream(zipFile)))) {
```

Comment:
should this line be

```
try (ZipArchiveInputStream archive = new ZipArchiveInputStream(new BufferedInputStream(new FileInputStream(zipFile)))) {
```

Reply:
I can't click on "[Apply Suggestion](https://github.com/microsoft/vscode-pull-request-github/blob/main/documentation/suggestAChange.md#accepting-a-suggestion)" button 😅. Will commit the suggestion soon.